Library/Homebrew/download_strategy.rb

```diff
@@ -388,6 +388,25 @@ def _fetch
   end
 end
 
+class CurlSourceForgeSecureMirrorDownloadStrategy < CurlDownloadStrategy
+  def fetch
+    if !ENV["HOMEBREW_NO_INSECURE_REDIRECT"].nil?
+      oldurl = @url
+      @url = https_mirror
+      ohai "Sourceforge HTTPS: #{oldurl}"
+      ohai "=> converted to #{@url}"
```

Comment:
`ohai` already prefixes its message by a fat arrow (`==>`), it might be better to use a symbol other than another arrow.

```irb
>> ohai "=> foo"
==> => foo
```